ztp/extra-manifests-builder/04-accelerated-container-startup/accelerated-container-startup.sh

```diff
@@ -57,6 +57,75 @@ restrictedCpuset() {
   return 1
 }
 
+getCPUCount () {
+  local cpuset="$1"
+  local cpulist=()
+  local cpus=0
+  local mincpus=2
+
+  if [[ -z $cpuset || $cpuset =~ [^0-9,-] ]]; then
+    echo $mincpus
+    return 1
+  fi
+
+  IFS=',' read -ra cpulist <<< $cpuset
+
+  for elm in "${cpulist[@]}"; do
+    if [[ $elm =~ ^[0-9]+$ ]]; then
+      (( cpus++ ))
+    elif [[ $elm =~ ^[0-9]+-[0-9]+$ ]]; then
+      local low=0 high=0
+      IFS='-' read low high <<< $elm
+      (( cpus += high - low + 1 ))
+    else
+      echo $mincpus
+      return 1
+    fi
+  done
+
+  # Return a minimum of 2 cpus
+  echo $(( cpus > $mincpus ? cpus : $mincpus ))
+  return 0
+}
+
+resetOVSthreads () {
+  local cpucount="$1"
+  local curRevalidators=0
+  local curHandlers=0
+  local desiredRevalidators=0
+  local desiredHandlers=0
+  local rc=1
```

Comment:
I think if I read this correctly, this means that if this function is a no-op (ie, no work was done because `[[ $curRevalidators -ne $desiredRevalidators || $curHandlers -ne $desiredHandlers ]]` evaluated to false), then we return 1 -> I think a no-op should return 0 so we don't count it as a failure?

Reply:
yes, good catch.